### PR TITLE
Docs: make naming consistent in the cloud storage & file system source

### PR DESCRIPTION
**Pull request by burnash**
This is a follow-up to #1804

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Cloud storage and filesystem
+# Cloud storage and file system
```
Comment (VioletM) [inline docs/website/docs/dlt-ecosystem/destinations/filesystem.md:1]: Hm, not sure we need to change this, especially considering we call a source `filesystem`, not `file_system`

image

File: docs/website/docs/dlt-ecosystem/verified-sources/filesystem/index.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 ---
-title: Filesystem & Buckets
-description: dlt-verified source for Filesystem & Buckets
-keywords: [readers source and filesystem, files, filesystem, readers source, cloud storage]
+title: Cloud storage and file system
```
Comment (VioletM) [inline docs/website/docs/dlt-ecosystem/verified-sources/filesystem/index.md:2]: Suggest we keep it the same as in the sidebar or change in both places :) 
```suggestion
title: Filesystem & cloud storage
```
image

**Review by VioletM — CHANGES_REQUESTED** [review]
Let's discuss if we should change the filesystem to file system. I don't think so :)

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md (hunk 1 above)
Comment (burnash) [reply docs/website/docs/dlt-ecosystem/destinations/filesystem.md]: ok, rolled the change back

File: docs/website/docs/dlt-ecosystem/verified-sources/filesystem/index.md (hunk 2 above)
Comment (burnash) [reply docs/website/docs/dlt-ecosystem/verified-sources/filesystem/index.md:2]: You're right, I've updated the sidebar so now it's also consistent with the destinations

File: docs/website/docs/dlt-ecosystem/verified-sources/filesystem/index.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
 ---
-title: Filesystem & cloud storage
-description: dlt-verified source for Filesystem & cloud storage
-keywords: [readers source and filesystem, files, filesystem, readers source, cloud storage]
+title: Cloud storage and file system
```
Comment (burnash) [inline docs/website/docs/dlt-ecosystem/verified-sources/filesystem/index.md:2]:
```suggestion
title: Cloud storage and filesystem
```

**Review by VioletM — APPROVED** [review]

**Pull request merged**